keyboards/converter/usb_usb/keymaps/via/keymap.c

```diff
@@ -0,0 +1,127 @@
+    KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,   KC_B,   KC_N,   KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RO,   KC_RSFT,              KC_UP,               KC_P1,   KC_P2,   KC_P3,   KC_PEQL,    KC_EXEC, KC_PSTE,
+    KC_LCTL, KC_LGUI, KC_LALT, KC_MHEN, KC_HANJ,         KC_SPC,         KC_HAEN, KC_HENK, KC_KANA, KC_RALT, KC_RGUI, MO(1),   KC_RCTL,     KC_LEFT, KC_DOWN, KC_RGHT,    KC_P0,            KC_PDOT, KC_PENT,    KC_FIND, KC_CUT
+    ),
+    [1] = LAYOUT_via(
+                      ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,
+    ______,           ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,               ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,
+    ______,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,  KC_F6,  KC_F7,  KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  ______,  ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  KC_UP,   ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,           ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  KC_LEFT, KC_DOWN, KC_RGHT, ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,                                    ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,               ______,              ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,          ______,         ______,  ______,  ______,  ______,   MO(2),  ______,  ______,      ______,  ______,  ______,     ______,           ______,  ______,     ______,  ______
+    ),
+    [2] = LAYOUT_via(
+                      ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,
+     RESET,           ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,               ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,
+    ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,  ______,  ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______, EEP_RST,  ______,  ______,           ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
```

Comment:
```suggestion
    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  EE_CLR,  ______,  ______,           ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
```

Reply:
Replaced with QK_CLEAR_EEPROM in latest push.